web-app/js/portal/details/LayerControlPanel.js

```diff
@@ -19,37 +19,50 @@ Portal.details.LayerControlPanel = Ext.extend(Ext.Container, {
             this.items.push(layerSelector, { xtype: 'spacer', height: 10 });
         }
 
-        this.items.push(this._newOpacitySlider());
+        this.opacitySlider = new Portal.common.LayerOpacitySliderFixed({
+            layer: this.layer,
```

Comment:
@dnahodil Agh yes i did understand you. It was needed for tests, to get a reference to this.dataCollection for the tracking